### DISCUSSION TITLE
Does audiogridder support raspberry Pi ARM CPU’s as a master

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#442] [Q&A]
Does audiogridder support raspberry Pi ARM CPU’s as a master

**Comment by CaBuffalo** [comment]
No, it doesn't from what I understand.

On Mon, Mar 22, 2021 at 12:53 PM bsjdsp ***@***.***> wrote:

> Does audiogridder support raspberry Pi ARM CPU’s as a master
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/442>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6ECIKC7A4XALSHPJF3TE6N4BANCNFSM4ZTY6M2A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by bsjdsp** [reply]
Oh, I hope it does support it. I mean the fact that it doesn’t process any audio just streaming I think it should not be too difficult to support. I really hope it gets supported I’d it’s not as I feel it would definitely make sense for my usage to support it.

**Comment by CaBuffalo** [comment]
The alternative is to use a used PC, don't need to spend too much.

See if Andreas will put it on the road map.

On Mon, Mar 22, 2021 at 2:05 PM bsjdsp ***@***.***> wrote:

> Oh, I hope it does support it. I mean the fact that it doesn’t process any
> audio just streaming I think it should not be too difficult to support. I
> really hope it gets supported I’d it’s not as I feel it would definitely
> make sense for my usage to support it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/442#discussioncomment-515157>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EPYEHZFF3LNND7SBDTE6WJ5ANCNFSM4ZTY6M2A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
There is no server support for linux right now. So unless you want to run a DAW on your pi, it is not supported.

Linux server support has been discussed a few times. We might have it at some point, but I'm not planning to work on this in the near future, as there is too many other things on the list.

**Reply by bsjdsp** [reply]
Does this mean it won’t be possible to use a raspberry Pi to run plugins from a windows or Mac server. That’s what I’m looking forward to doing.

**Comment by CaBuffalo** [comment]
Yes, you can use Pi as DAW and other PC/MAC as slave server.

I thought you are using Pi as your slave server.

I have no experience on Raspberry Pi. Wondering how it can host a DAW to
processing audio.

On Tue, Mar 23, 2021 at 11:53 AM bsjdsp ***@***.***> wrote:

> Does this mean it won’t be possible to use a raspberry Pi to run plugins
> from a windows or Mac server. That’s what I’m looking forward to doing.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/442#discussioncomment-519392>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BV6JMEKTT6CWV3FITTFDPT3ANCNFSM4ZTY6M2A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by bsjdsp** [reply]
Are you saying the raspberry Pi can install and run the audio gridder plugin to stream plugins from a server??

**Comment by CaBuffalo** [comment]
AG has the Linux version client plugin. As long as you have AG installed on
your Pi, yes, you can give it a try.

On Tue, Mar 23, 2021 at 1:50 PM bsjdsp ***@***.***> wrote:

> Are you saying the raspberry Pi can install and run the audio gridder
> plugin to stream plugins from a server??
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/442#discussioncomment-519834>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FHYL53JFXHVCMU5NDTFD5IVANCNFSM4ZTY6M2A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by bsjdsp** [reply]
Well it depends because audio grosser would have to also support ARM based cpu architectures In order for it to run, I have lots of Linux plugins but not all of them can be installed or run on ARM based computers, I haven’t found any documentation that audio gridder works in a Pi not have I seen anyone using it on a Pi yet. I don’t have a Pi yet to try so I guess I’ll see how it goes over time. Maybe one someone knows. I wish audio gridder had better documentation on these things

**Reply by apohl79** [reply]
The is no precompiled arm build for linux. You will have to compile it yourself.

**Comment by Clintcinnati** [comment]
Traction Waveform 11.5 free (and the OEM version) and Pro all claim to work fine on Pi, and it’s a great DAW now, actually inspiring and fun-
I have it installed on my iMac and am using another iMac as audiogridder server and Waveform works fine with audiogridder.

Waveform is a little weird in that all versions, Mac, Linux (including Pi), and Windows look exactly the same, they don’t utilize the host OS menus etc, so you have to watch a tutorial or read the manual (there is a MANUAL!!) to get started, but it’s very logical and intuitive.  Being free and currently supported, and with active Raspberry support, it might be a good DAW for Raspberry.
